mteb/load_results/load_results.py

```diff
@@ -6,14 +6,18 @@
 import subprocess
 from collections import defaultdict
 from pathlib import Path
+from typing import Dict, List, Sequence
 
-import mteb
+from mteb.abstasks.AbsTask import AbsTask
+from mteb.load_results.mteb_results import MTEBResults
 from mteb.model_meta import ModelMeta
 
 logger = logging.getLogger(__name__)
 MODEL_NAME = str
 REVISION = str
 
+RESULTS = Dict[MODEL_NAME, Dict[REVISION, List[MTEBResults]]]
```

Comment:
for 3.8 you have to use the typing.Dict, but you can remedy this for function by importing from __future__. However this does not work outside of a function/class. So here using dict would cause an error, but in python version >=3.9.